tools/pagerduty-migrator/migrator/report.py

```diff
@@ -47,7 +47,7 @@ def format_escalation_policy(policy: dict) -> str:
 
 
 def format_integration(integration: dict) -> str:
-    result = integration["service"]["name"] + " - " + integration["name"]
+    result = "{} - {}".format(integration["service"]["name"], integration["name"])
```

Comment:
_total nit_
```suggestion
    result = f"{integration['service']['name']} - {integration['name']}"
```

Reply:
I kinda prefer `format` here, seems to be easier to understand what the result would be from a quick look. Probably a taste thing 😁